### PR TITLE
Task/add collateral bk

### PR DESCRIPTION
**Pull request by lbk003**
Misc clean up of the files. This is not normally a recommended way of doing PRs - more an exception to get minor things in place before pushing to PyPi. I have left comments on the files of what the change is.

### CODE REVIEW

File: banidb/banidb.py [hunk 1]
```diff
@@ -103,7 +105,6 @@ def shabad(shabad_id, larivaar=False):
         for verse in json_blob['verses']:
             line = {'verse_id': verse['verseId']}
             if larivaar is True:
-                shabad_id = f"{shabad_id}lari"
```
Comment (lbk003) [inline banidb/banidb.py:106]: This was appending `lari` for every verse, so removed it.

File: banidb/history.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import pickle
 import os
+import pickle
```
Comment (lbk003) [inline banidb/history.py:2]: Ran `isort` on the file

File: tests/test_banidb.py [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
+import requests
 from requests_mock.mocker import Mocker
 
+import banidb
+
 url = 'https://api.banidb.com/v2'
 
 
+@pytest.fixture
+def clear_cache():
```
Comment (lbk003) [inline tests/test_banidb.py:14]: For deterministic output, cache should be cleared before running the test. There should be additional tests for testing the caching.

File: tests/test_history.py [hunk 4]
```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
-from banidb.history import LRUCache
 import pytest
 
+from banidb.history import LRUCache
```
Comment (lbk003) [inline tests/test_history.py:4]: Ran `isort` on the file.

File: tests/test_banidb.py [hunk 5]
```diff
@@ -159,50 +132,8 @@ def test_shabad_ok_larivaar_true(requests_mock: Mocker):
     }
 
 
-def test_shabad_not_ok_larivaar_false(requests_mock: Mocker):
```
Comment (lbk003) [inline tests/test_banidb.py:162]: The test payload was repeated many times over, so consolidated it.

**Review by MySmilingTurban — APPROVED** [review]

**Review by tsingh777 — APPROVED** [review]

**Pull request merged**